specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json

```diff
@@ -3452,15 +3461,15 @@
         "sizeInGB": {
           "format": "int32",
           "description": "Size of the persistent disk in GB",
-          "maximum": 50.0,
-          "minimum": 0.0,
+          "maximum": 50,
```

Comment:
Please don't include unnecessary changes in the PR, as well as the response status code above.

Reply:
So this is actually changed because I ran the prettier - it is an automatic change. Do you still think I should revert?

Reply:
I think so. It is hard to track when one PR introduces more than one proposals.